### ISSUE TITLE
Add Graph

### ISSUE THREAD

**Issue by yangshun** [yangshun/lago#12]

**Comment by tgrander** [comment]
I am wondering if you have any ideas on how you would prefer this data structure to be implemented. 

For example, basic implementation of Graph include very simple methods such as this: 

`
addVertex(vertex) {
    this.vertices.push(vertex);
    this.edges[vertex] = [];
  }
`

This method for adding a vertex would assume that all new vertices are numbers and are all unique without doing any error checking on the values being passed as vertex. 

Basically, what is the level of complexity you hope these methods to contain? i.e can vertices only be numbers? Or should vertices be able to be any value?

**Comment by akthecoders** [comment]
Can i will work on the graph Data Structure?

**Comment by yangshun** [comment]
Go ahead

**Comment by yangshun** [comment]
Closed via 95b9257
